contracts/contracts/FleekERC721.sol

```diff
@@ -26,6 +26,13 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
 
     event NewAccessPoint(string apName, uint256 indexed tokenId, address indexed owner);
     event RemoveAccessPoint(string apName, uint256 indexed tokenId, address indexed owner);
+
+    event ChangeAccessPointAutoApprovalSettings(
```

Comment:
Do we need this event? Maybe it's something we can add later if we want to keep track?

Reply:
Just for subgraph.